src/node/internal/internal_http_server.ts

```diff
@@ -300,6 +238,32 @@ export class Server
     // TODO(soon): Revisit this once we implement net.Server
     return this;
   }
+
+  get listening(): boolean {
+    return this.port != null;
+  }
+
+  address(): string | AddressInfo | null {
+    if (this.port == null) return null;
+    return { port: this.port, family: 'IPv4', address: '127.0.0.1' };
```

Comment:
Just noting... we need to make sure we adequately document the fact that an `http.Server` only supports the `listen(...)` variant where we pass in a port to listen. Specifying the hostname, family, etc won't be supported.

We also need to make sure we document that `address().address` will always be hardcoded to `127.0.0.1`